fhir-core/src/test/java/com/ibm/fhir/core/test/mains/DateTestMain.java

```diff
@@ -6,22 +6,6 @@
 
 package com.ibm.fhir.core.test.mains;
 
-import static com.ibm.fhir.core.FHIRUtilities.*;
-import static org.testng.AssertJUnit.*;
-
-import java.util.Arrays;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.List;
-
-import javax.xml.datatype.DatatypeFactory;
-import javax.xml.datatype.Duration;
-import javax.xml.datatype.XMLGregorianCalendar;
-
-import org.testng.annotations.Test;
-
-import com.ibm.fhir.core.FHIRUtilities;
-
 public class DateTestMain {
```

Comment:
I think this is a class to revisit.  Not to remove the content.

Reply:
I don't think we need this class, so removed.